### DISCUSSION TITLE
If I click with uc_click() it opens in new tab so is it detectable?

### DISCUSSION THREAD

**Discussion by PankajSavaliya** [seleniumbase/SeleniumBase#2905] [General]
I have used `self.uc_click()` to click, and it opens in a new tab. Is this detectable in UC mode with multi-threading? How can I restrict it from opening in a new tab? I have read many issues and noticed your recommendation against using multi-tab in UC mode.

I apologize for all these questions and greatly appreciate your quick response. @mdmintz

**Comment by mdmintz** [comment]
`uc_click()` won't open a new tab unless the element you're clicking on does that automatically when it gets clicked.

High traffic from an IP Address can make you detectable, so take caution when multi-threading. You'll have to experiment, as there's no definite answer.

**Reply by PankajSavaliya** [reply]
If a new tab opens when clicked, that's okay. However, what should I take care of with the new tab in UC mode so that I can handle it even when a new tab opens except ip? @mdmintz

**Reply by mdmintz** [reply]
This is not a specific question. Experiment on your own and see what works.
